src/OSPSuite.SimModel/src/Simulation.cpp

```diff
@@ -900,7 +905,8 @@ void Simulation::SetObserverSensitivityValues(int index, const double time, doub
 		int observerIdx, variableIdx, parameterIdx;
 		for (observerIdx = 0; observerIdx<observersSize; observerIdx++)
 		{
-			if (_observers[observerIdx]->IsConstantDuringCalculation())
+			auto observer = _observers[observerIdx];
```

Comment:
auto? what does that  mean? SAme as var?

Reply:
yep. was introduce with C++2011 (or 2014)